CHANGES_UNRELEASED.md

```diff
@@ -9,3 +9,13 @@
 - Android: Added ability to rekey the database via `rekeyDatabase`. [[#2228](https://github.com/mozilla/application-services/pull/2228)]
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.43.1...master)
+## FxA Client
+
+### What's new
+
+* Android: BREAKING CHANGE: `migrateFromSessionToken` now reuses the existing 'sessionToken' instead of creating a new session token.
```

Comment:
nit: I believe our convention is to have a separate `### Breaking Changes` heading rather than labelling breaking changes inline like this.